src/screens/WalletConnect/Home/WalletConnectHome.js

```diff
@@ -100,33 +99,23 @@ function WalletConnectHome() {
   // Note: in order to achieve multicolumn layout, we group n normal items into one list row item.
   const renderListRow = (items: WalletConnectCmsApp[]) => <ListRow>{items.map(renderItem)}</ListRow>;
 
-  const openUrl = async (url: string | null) => {
+  const openAppUrl = (url: string | null) => {
```

Comment:
Seems like we no longer shoe anything if url is falsy, is that intended?

Reply:
Done.